xline/src/request_validation.rs

```diff
@@ -180,9 +181,10 @@ impl RequestValidator for AuthRoleGrantPermissionRequest {
 }
 
 /// Error type in Validation
-#[derive(Error, Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Default, Error, Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub enum ValidationError {
     /// Key is not provided
+    #[default]
```

Comment:
Why do we need a default value for an error type?

Reply:
This derive is also used in tests. Now I refactored it to conditionally derive in tests